### ISSUE TITLE
Media CSS

### ISSUE THREAD

**Issue by Apogate** [spootTheLousy/saguaro#247]
Revisit issues raised in #243.

**Comment by RePod** [comment]
***TL;DR:***
 - Use sane CSS classes to be combined in any way a developer/theme creator may want.
 - For example, separate `media` and `multi` classes selected as `.media.multi`/`.multi.media`.

Also in recent developments of seeing what we can handle, ~260 media files is the current artificial limit with overall short file names. However, there is no imposed wrap on the files so it gets out of hand quickly. So in addition to the above:

 - Force the files container to a limited size and use scroll bars
   - Potentially custom/skinned scrollbars to be flush with the page
 - Use `text-overflow:ellipsis` for filenames (or all fields) to limit the width further
   - Stick the filename in a tooltip
